words.txt

```diff
@@ -1,22 +1,328 @@
 ﻿
-﻿µsr
-﻿aa
+0005k
+000us
+00am
+00mm
+0138xx
+0167x
+0358xx
+0434xx
+05k
+0776xx
+0c
+0e
+0mm
+0x02
+0x1
+1000mm
+100c
+100k
+100kn
+100ma
+100mm
+100ms
+100px
+100x
+1080p
+10hz
+10k
+10mg
+10ms
+10th
+10v
+115kb
+120s
+12xx
+132s
+14gb
+14t
+15s
+16mb
+16s
+16th
+17t
+180hz
+18th
+1920x1200
+1d
+1gb
+1hz
+1k
+1mm
+1mot
+1pm
+1s
+1st
+1t
+1v
+200kv
+2023a
+20analysis
+20chopper
+20crate
+20gb
+20handover
+20k
+20lifter
+20mk
+20modules
+20monitoring
+20ms
+20px
+2100n
+218s
+21st
+232c
+24gb
+24th
+25hz
+25k
+25v
+260mk
+2661n
+26th
+26x
+28px
+28th
+29th
+2abs
+2d
+2eres
+2fcomputing
+2ficpdiscussions
+2fisis
+2flabview
+2hz
+2k
+2m
+2mm
+2ms
+2nd
+2t
+2x
+3001p
+300k
+300mk
+3021b
+3022b
+30gb
+30k
+30mk
+30seconds
+30th
+32bit
+33220a
+360multip
+3_eurotherms
+3a
+3b
+3c
+3d
+3e
+3f
+3rd
+3rdparty
+3t
+400mb
+400ms
+40k
+40mk
+4104b
+4146x
+42p
+4454ecf8f3f1d0bc324af6cf134c45ef5ef39ad0
+4519x
+45am
+46dd4e
+4gb
+4k
+4m
+4pm
+4t
+4th
+500mb
+500mm
+500ms
+50c
+50gb
+50hz
+50kn
+50m
+50mm
+53220a
+532s
+554a
+595hz
+599hz
+5_axis
+5gb
+5ghz
+5gs
+5ma
+5mm
+5ms
+5s
+5t
+5v
+600a
+600hz
+600s
+600w
+606hz
+64bit
+64d
+64s
+6517b
+6dp
+6gb
+6k
+6kv
+6th
+75000a
+750a
+75v
+7k
+7t
+7u40
+7zip
+800c
+8018r
+80k
+80x80
+8gb
+8h
+8k
+8t
+8u20
+8v
+9000v
+9001x
+9am
+9t
+_0x
+_3_eurotherms
+__blank
+__blank__
+__file__
+__line__
+__note
+__rep__
+_adjust
+_adroja
+_and_
+_apache
+_base
+_block
+_blocks
+_both_
+_c_
+_casnooper
+_chosen
+_configuration
+_configuration_
+_copy_
+_crisp_
+_dbd
+_detector_
+_dom
+_especially_
+_fan
+_for
+_frames
+_front
+_glossary
+_height
+_home_
+_i
+_ibex_
+_in
+_insert
+_instname_
+_inter_
+_ioc_
+_is_
+_level_
+_lib
+_libs
+_loq
+_maven
+_mevents
+_might_
+_minimally
+_move
+_move_
+_moving1
+_my_instrument_name_
```

Comment:
It looks like the previous library split this into `my`, `instrument` and `name` then checked all of them separately, which is much better as it would lead to less errors